src/lib/packager/clients/pkg_finish.rb

```diff
@@ -284,15 +290,18 @@ def sync_target_sources
       Pkg.TargetInitialize(Installation.destdir)
     end
 
-    # Set libzypp configuration to install the minimal amount of packages
+    # Set package manager configuration to install the minimal amount of packages
     #
     # @see Yast::Packager::CFA::ZyppConf#set_minimalistic!
-    def set_minimalistic_libzypp_conf
-      log.info("Setting libzypp configuration as minimalistic")
-      config = Packager::CFA::ZyppConf.new
-      config.load
-      config.set_minimalistic!
-      config.save
+    def set_minimalistic_pkg_conf
+      PKG_MAPPING.each do |pkg, cfa|
+        next unless File.file?(File.join(Installation.destdir, cfa::PATH))
+        log.info("Setting #{pkg} configuration as minimalistic")
```

Comment:
Nitpicking: if use `#{cfa::PATH}` here then we do not need the `pkg` values at all and we could replace the Hash with an Array.... :grin: